### DISCUSSION TITLE
TailwindCSS class name to object

### DISCUSSION THREAD

**Discussion by nagibmahfuj** [nativewind/nativewind#479] [Ideas]
Hi, is there any way in this package to convert tailwindcss class names to css objects? Like below:
`flex-grow flex justify-center h-full items-center`
 Converted:
`{display: "flex",flexGrow: 1,justifyContent: "center",alignItems: "center",height: "100%"}`
Any kind of utility function that can convert in to object.

**Comment by lstellway** [comment]
You can require / use the [`twrnc`](https://github.com/jaredh159/tailwind-react-native-classnames) dependency directly. 
_(learned about it from a [recent commit](https://github.com/marklawlor/nativewind/commit/4d308c9118a413a171545ce4275e2216b35c2010))_

**Comment by YKSing** [comment]
I did this to get the converted style object.
```
const tw = (className: string | (string | false)[]) => {
  const StyledComponent = styled(() => null);
  const component = <StyledComponent />;
  const render = component.type.render({
    className: Array.isArray(className)
      ? className.filter(Boolean).join(' ')
      : className,
  });
  return render.props.style;
};
```

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
